### DISCUSSION TITLE
Smart money concept (smc)

### DISCUSSION THREAD

**Discussion by ArtemHachaturov** [DaveSkender/Stock.Indicators#1210] [Ideas]
Please share your opinion about this concept and the possibility of adding it to your indicator database in the future.

-Fair Value Gap (FVG)
-Swing Highs and Lows
-Break of Structure (BOS) & Change of Character (CHoCH)
-Order Blocks (OB)
- Breacking  Block (BB)
- Mitigation Block (MB)
-Liquidity
-Previous High And Low
-Sessions
-Retracements

Developed algorithms and tools will have practical utility, aiding traders in making more informed decisions and enhancing their trading outcomes

**Comment by DaveSkender** [comment]
I’d have to study this strategy before forming an opinion.  Generally, I do like ones that use a wide range of indicators, like this one.

Related to:
- #1201
